src/vstart.sh

```diff
@@ -663,12 +664,17 @@ start_mgr() {
 EOF
 
 	ceph_adm config-key set mgr/dashboard/$name/server_port $MGR_PORT
-	DASH_URLS+="http://$IP:$MGR_PORT/"
+	DASH_URLS+=" http://$IP:$MGR_PORT/"
```

Comment:
Why the blank before the values? This looks "unusual".

Reply:
Because in the end that variable is going to be echoed showing the list of URLs that can be used to access the dashboard